pom.xml

```diff
@@ -165,6 +165,10 @@
           <scope>test</scope>
       </dependency>
       <dependency>
+          <groupId>org.apache.httpcomponents</groupId>
+          <artifactId>httpclient</artifactId>
```

Comment:
This will force that library to be bundled with _every plugin_ in your reactor. Sounds wrong.

(Generally speaking, you should avoid `dependencies` in a POM parent; only use `dependencyManagement` and have each component declare those `dependencies` it actually needs. I guess it does not harm much to add `test`-scoped dependencies, though even those are probably undesirable. `junit:junit` in particular, @ line 148 above, is redundant.)

Reply:
Removed junit.  I think declaring test is something makes it easy for sub-modules. Changed httpclient scope to test.